RELEASING/README.md

```diff
@@ -74,6 +90,8 @@ Then you can generate other derived environment variables that are used
 throughout the release process:
 
 ```bash
+    # Replace FULLNAME with your PGP key name for Apache
+    export FULLNAME="YOURFULLNAMEHERE"
```

Comment:
We should probably prefix all release related env variables with `SUPERSET_` as per @etr2460 recommendation.

Reply:
Changed, replaced it on the initial setup also